src/components/ChangePlan/ChangePlan.test.js

```diff
@@ -442,4 +453,50 @@ describe('ChangePlan component', () => {
       );
     });
   });
+  it('should redirect to UpgradeForm when user has exclusive plan', async () => {
+    // Arrange
+    const paths = [{}];
+    const dependencies = {
+      appSessionRef: {
+        current: {
+          userData: {
+            user: {
+              plan: {},
```

Comment:
You could add the exclusive plan here and let the planservice do de mapping right? Perhaps not, you tell me. 😄

Reply:
In don't understand how to do it, tomorrow I will call you 📲